### DISCUSSION TITLE
Is it possible to use the official nord vim theme in NvChad?

### DISCUSSION THREAD

**Discussion by luh-05** [NvChad/NvChad#2658] [Q&A] [answered]
I really like the official Nord vim theme: https://www.nordtheme.com/ports/vim
Is there any way to use this theme in NvChad? If so how?

Also the bar looks a bit wonky for some reason, with some stuff getting shifted up slightly: 
![image](https://github.com/NvChad/NvChad/assets/60015267/af68b76e-3bc4-4cef-8a44-e262a7611b1d)
Any help in fixing that would also be appreciated!

**Comment by siduck** [comment, marked as answer]
that bar thing is your terminal issue :/

you cant use external themes in nvchad. just use our nord port and improve it if you want

**Reply by luh-05** [reply]
Thanks for the answer!
Then I'll play around with the in-built nord theme at some point when I find time.

Also for anyone having the same problem with the bar: Try changing your font, worked for me at least
